_posts/2017-04-12-managing-technical-debt.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Managing Technical Debt
+description: "Quite obvious plan to manage a technical debt."
+author: bogdan
+modified: 2014-12-24
+tags: [code review, agile, process, qa]
+categories: [process]
+image:
+    feature: posts/tech_debt/title.jpg
+---
+
+# WHAT IS “TECHNICAL DEBT”?
+
+Martin Fowler [has a great explanation](https://martinfowler.com/bliki/TechnicalDebt.html) of this term and the problem behind it.  Many startups and new projects use quick-and-dirty solutions to achieve their goals as soon as possible, which in some point makes project complicated, slow, hard to change or implement a new features and full of bugs. I want to talk about most usual issues and how manage most effectively from my point of view.
```

Comment:
...Many startups and new projects use quick-and-dirty solutions to achieve their goals as soon as possible, which [at some point, in some cases, in some instances???] overcomplicates a project, slows it down, makes it hard to change or implement a new features and full of bugs.  I'd like to bring up the most common issues and how to manage them in the most effective way, from my point of view.